### ISSUE TITLE
"hidden" condition does not handle "StaleElementReferenceException" correctly

### ISSUE THREAD

**Issue by dmak** [codeborne/selenide#224]
On some reason `hidden` condition does not handle `StaleElementReferenceException`:
```
com.codeborne.selenide.ex.ElementShould: Element should be hidden {By.xpath: //table[@id='main']/tbody/tr[0]}
Element: 'StaleElementReferenceException: Element is no longer attached to the DOM'
	at com.codeborne.selenide.impl.AbstractSelenideElement.checkCondition(AbstractSelenideElement.java:512)
	at com.codeborne.selenide.impl.AbstractSelenideElement.should(AbstractSelenideElement.java:476)
	at com.codeborne.selenide.impl.AbstractSelenideElement.invokeShould(AbstractSelenideElement.java:299)
	at com.codeborne.selenide.impl.AbstractSelenideElement.dispatchSelenideMethod(AbstractSelenideElement.java:182)
	at com.codeborne.selenide.impl.AbstractSelenideElement.dispatchAndRetry(AbstractSelenideElement.java:92)
	at com.codeborne.selenide.impl.AbstractSelenideElement.invoke(AbstractSelenideElement.java:67)
	at com.codeborne.selenide.impl.WaitingSelenideElement.invoke(WaitingSelenideElement.java:18)
```
The stack trace is not complete so I don't know if it was correctly handled in [`Condition#74`](https://github.com/codeborne/selenide/blob/master/src/main/java/com/codeborne/selenide/Condition.java#L74), or it happened because of the same reason described in issue #130.

Selenide v2.22.

**Comment by asolntsev** [comment]
@dmak Yes, this issues seems to be the same as #130. 
Can you still reproduce the issue? Could you send a sample code that would help me reproduce the problem?

**Comment by dmak** [comment]
At the moment it does not appear on v2.23. Was there anything related to that fixed in 2.23?

**Comment by asolntsev** [comment]
@dmak No, I guess not. Probably it's an issue of browser speed.
Can we close the issue now?
